packages/airnode-protocol/package.json

```diff
@@ -29,21 +29,21 @@
     "write-example-env-file": "hardhat run scripts/write-example-env-file.ts"
   },
   "devDependencies": {
-    "@api3/chains": "^4.4.0",
+    "@api3/chains": "^4.5.1",
     "@nomiclabs/hardhat-ethers": "^2.2.3",
-    "@nomiclabs/hardhat-etherscan": "^3.1.7",
+    "@nomiclabs/hardhat-etherscan": "^3.1.8",
```

Comment:
hardhat-etherscan [is deprecated](https://github.com/NomicFoundation/hardhat/blob/0b6ac0fa1b19f2db452cce02336e6db2082ade58/packages/hardhat-etherscan/README.md), created https://github.com/api3dao/airnode/issues/1950 for the migration